mkosi/burn.py

```diff
@@ -12,6 +12,9 @@ def run_burn(args: Args, config: Config) -> None:
     if config.output_format not in (OutputFormat.disk, OutputFormat.esp):
         die(f"{config.output_format} images cannot be burned to disk")
 
+    if not args.cmdline:
+        die("Please specify a device to burn the image to", hint="For example /dev/sdb")
```

Comment:
Our userbase will most likely not fall into the trap, but I'd rather not name a device that has chance of actually existing, how about `/dev/sdz`, which should be safe if there's not too many drives (under the assumption somebody sitting in front of machine with this many drives knows what they are doing) or `/dev/nvme1n1`, since I think the first number is a namespace and those are not in use in the wild.

Reply:
Even better (though a bit more verbose) might be `/dev/disk/by-id/usb-foobar` which I have come to use since its even harder to mess those up (and I do not get confused when switching between laptop and desktop with different disk types/counts)